Source/Classes/OverlayContainerViewController.swift

```diff
@@ -243,53 +237,30 @@ open class OverlayContainerViewController: UIViewController {
         overlayTranslationContainerView.pinToSuperview()
         overlayTranslationContainerView.addSubview(overlayTranslationView)
         overlayTranslationView.addSubview(overlayContainerView)
-        overlayTranslationView.translatesAutoresizingMaskIntoConstraints = false
-        overlayContainerView.translatesAutoresizingMaskIntoConstraints = false
-        setOverlayPosition()
-        setOverlayConstraints()
+        setupOverlayConstraints()
         loadTranslationController()
     }
 
-    private func setOverlayPosition() {
-        var translationConstraints: [NSLayoutConstraint] = [
-            overlayTranslationView.leftAnchor.constraint(equalTo: overlayTranslationContainerView.leftAnchor),
-            overlayTranslationView.rightAnchor.constraint(equalTo: overlayTranslationContainerView.rightAnchor)
-        ]
+    private func setupOverlayConstraints() {
+        overlayTranslationView.pinToSuperview(edges: [.bottom, .left, .right])
+        overlayContainerView.pinToSuperview(edges: [.left, .top, .right])
 
-        var overlayConstraints: [NSLayoutConstraint] = [
-            overlayContainerView.topAnchor.constraint(equalTo: overlayTranslationView.topAnchor),
-            overlayContainerView.leftAnchor.constraint(equalTo: overlayTranslationView.leftAnchor),
-            overlayContainerView.rightAnchor.constraint(equalTo: overlayTranslationView.rightAnchor)
+        landscapeLayoutConstraints = [
+            overlayTranslationView.topAnchor.constraint(equalTo: overlayTranslationContainerView.topAnchor),
+            overlayContainerView.bottomAnchor.constraint(equalTo: overlayTranslationView.bottomAnchor)
         ]
```

Comment:
When active, these constraints complete the edges not pinned by the constraints created on line `245-246` — so when they are active all the views in the hierarchy have the same size.